src/test/kotlin/org/rust/cargo/project/RustProjectSettingsServiceTest.kt

```diff
@@ -35,10 +35,8 @@ class RustProjectSettingsServiceTest : LightPlatformTestCase() {
               <option name="externalLinterArguments" value="--no-default-features" />
               <option name="macroExpansionEngine" value="DISABLED" />
               <option name="runExternalLinterOnTheFly" value="true" />
-              <option name="runRustfmtOnSave" value="true" />
               <option name="toolchainHomeDirectory" value="/" />
               <option name="useOffline" value="true" />
-              <option name="useRustfmt" value="true" />
```

Comment:
Since we don't drop these fields from the state itself, let's keep them here as well